client/app/reducers/index.js

```diff
@@ -0,0 +1,6 @@
+import { combineReducers } from 'redux'
+import { NavReducer } from './nav'
+
+export const RootReducer = combineReducers({
+  nav: NavReducer
```

Comment:
Why do we want to rename the reducer? I think it would be better to use full name here so we know with what we are dealing with.

```javascript
export const RootReducer = combineReducers({
  NavReducer
})
```